tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -310,6 +331,42 @@ private async Task<AuthenticationResult> RunTestForUserAsync(LabResponse labResp
             return result;
         }
 
+        private Tuple<IPublicClientApplication, HttpSnifferClientFactory> getPCA(LabResponse labResponse, bool directToAdfs, bool multiCloudSupport)

```

Comment:
In test projects you can use the fancier tuples from C# (in MSAL source code you cannot because of .net 45)

```csharp
 private Tuple<IPublicClientApplication pca, HttpSnifferClientFactory httpFactory> getPCA() { ... } 

// later in the code instead of var tuple = getPCA(labResponse, false, true);
(IPublicClientApplication Pca, HttpSnifferClientFactory HttpFactory) tuple = getPCA(labResponse, false, true);
tuple.Pca.AcquireToken(...)

```